src/formatDate.js

```diff
@@ -8,7 +8,60 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateParts = date.split(fromFormat[3]);
+  const obj = {};
+  const tab = [];
```

Comment:
Use more descriptive variable names. Use descriptive names for variables like for example `dateObj`, `formattedDateParts`.